### ISSUE TITLE
DESIGN - As a user, I want gitcoin.co/is_not_a_shitcoin to exist, so I can understand why Gitcoin did not do a token / token sale.

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#1443]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
DESIGN - As a user, I want gitcoin.co/is/not/a/shitcoin to exist, so I can understand why Gitcoin did not do a token.

### Why Is this Needed?

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:
There are so many ICOs and I might think Gitcoin is one of them if I was none the wiser.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: 
Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
Doesnt exist

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Exists

I have a document that describes the content of the page.  If you are approved to work on the task I will share it with you.

### Bounty

This bounty is for *design* of said page.  Please feel free to use anything in [this repo](https://github.com/gitcoinco/creative) especially the [brand guide](https://github.com/gitcoinco/creative/tree/master/Brand).  Also, https://gitcoin.co/mission and https://gitcoin.co/results are a good place to draw some inspiration.

If you want to work on this bounty, please do two things
1. Post a link to your portfolio
2. Request access to the above document.

Design will have to be responsive.

I would very much like to have a first draft of this page by 20 days from now, so please do not apply unless you can make that happen.

Thanks, looking forward to working with you.
- Kevin

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 4 weeks, 1 day from now. Please review their questions below:


1. [groundRachel](https://gitcoin.co/profile/groundRachel) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/1443/607?mutate_worker_action=approve&worker=groundRachel) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/1443/607?mutate_worker_action=reject&worker=groundRachel))_. 	
 * Q: ewrwerfsdf

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 4 weeks, 1 day from now. Please review their questions below:


1. [junagao](https://gitcoin.co/profile/junagao) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/1443/607?mutate_worker_action=approve&worker=junagao) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/1443/607?mutate_worker_action=reject&worker=junagao))_. 	
 * Q: Hello Kevin,

I'd like to start working on this project. Here's my portfolio: https://junagao.com/
Also, as specified on step 2. to request access to the above document, which document is it?

Many thanks,
Juliane

**Comment by owocki** [comment]
@junagao excited to work with you! [here](https://medium.com/gitcoin/why-gitcoin-didnt-do-a-utility-token-ico-and-neither-should-you-ec942733dd6b) is the article upon which i'd like this page designed.  

@Legitimate good to meet you.  i decided to approve the other bounty hunter since they were first and had a portfolio i can check out.  if you send me a message on the gitcoin slack, i'm sure i have other design work we can collab on tho!

**Comment by junagao** [comment]
@owocki can you please explain me more in details the briefing of the project? I'm a bit confused. Is the article you sent - the text as it is - the actual content of the page? can I message you on slack?

**Comment by owocki** [comment]
@junagao what questions do you have?  the article i sent you is a content piece that describes the major points about why we did not launch a token.  i'm counting on you to design a page that makes those points visually.

**Comment by owocki** [comment]
yes pls msg me on slack

**Comment by gitcoinbot** [comment]
@Legitimate Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by gitcoinbot** [comment]
@Legitimate Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by gitcoinbot** [comment]
@junagao Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by vs77bb** [comment]
Hi @junagao do you have any questions or progress to share? Hope you're doing well. If you need additional inspiration, we have also drafted [this article](https://medium.com/p/f24af0b54ded/edit) along the same lines. 

@owocki, perhaps we might consider a tamer URL once this goes live 🙂

**Comment by junagao** [comment]
Hi @owocki and @vs77bb, how can I submit a WIP PR? I'd like to send a preview so you can comment on. I'd like to have some feedback to whether or not I'm in the right direction.

Also, in which file format do you usually work for the design? .sketch or .ai?

Thank you,
Juliane

**Comment by gitcoinbot** [comment]
@junagao Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by thelostone-mc** [comment]
@junagao Ah sorry about this ! Everyone got a lil busy this week 😅 
You can submit a WIP PR to https://github.com/gitcoinco/creative
That's where all our design assets reside.

We are good with both `sketch` and `ai`

cc @PixelantDesign whos's our design lead on this ^_^

You can either attach screenshots here or in the PR whichever you prefer

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

@junagao due to inactivity, we have escalated [this issue](https://gitcoin.co/issue/gitcoinco/web/1443/607) to Gitcoin's moderation team. Let us know if you believe this has been done in error!

* [x] warning (3 days)
* [x] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by junagao** [comment]
Hi, I submitted a WIP PR yesterday. Please take a look.

Many thanks,
Juliane

**Comment by owocki** [comment]
thanks for submitting the WIP PR.

i think the light green is def overused.   is it possible to do a page in the style of the /about or /mission or /results page?

**Comment by gitcoinbot** [comment]
@junagao Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by thelostone-mc** [comment]
@junagao in case you missed the ping from @owocki ^

**Comment by junagao** [comment]
yes, sure. i will make the changes and submit a new PR later.

@thelostone-mc thanks :)

**Comment by gitcoinbot** [comment]
@junagao Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by owocki** [comment]
hows it going?

**Comment by owocki** [comment]
@junagao are you still interested in this issue?  please let me know if not so i can return it to an open state and have someone else work on it

**Comment by junagao** [comment]
@owocki yes, still interested! sorry my delay to answer. got caught up with other work.

Here's how it looks so far. Let me know what you think.

![gitcoin-03](https://user-images.githubusercontent.com/615616/42624603-08029e44-85c6-11e8-8733-56fb2a74b52f.png)

**Comment by owocki** [comment]
thanks! i think the light green is def overused. is it possible to do a page in the style of the /about or /mission or /results page?

**Comment by junagao** [comment]
Sorry, I think I didn't understand the first time.
I was also doing another version meantime. This one is complete with all the content.

![is-not-a-shit-coin](https://user-images.githubusercontent.com/615616/42679390-11429a1a-8682-11e8-8f7a-483a9bb645b9.png)

**Comment by owocki** [comment]
thanks for taking another pass!

i wonder if we can eliminate a lot of the text and just focus on the core ideas (via titles / iconography), like we do on the gitcoin.co/mission page

**Comment by junagao** [comment]
Yes, when I first saw it I thought there were a lot of text. But what are the core ideas?
Could you help me with that? Please, let me know how you want the text to be.

**Comment by thelostone-mc** [comment]
@vs77bb @scottydelta if in case you guys can help out  ^ 

@junagao thanks for being super accommodating with the changes 😅 
You are awesome ❤️

**Comment by gitcoinbot** [comment]
@junagao Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by vs77bb** [comment]
Hi @junagao here's my take on core ideas:

- It is still early days in blockchain, including tokens / tokenization. 
- Gitcoin didn't want to launch with a token before finding product-market fit, and doesn't think it's the morally right move. .
- There are a number of other options to explore in terms of capital in web3 (great job highlighting these!)
- Founders should consider these other options, and think before doing an ICO. Make the right chess move with your time. 

I think if you reduce 50-70% of the text while keeping these core ideas, you'll be in good shape! Feel free to DM me with questions on Slack @vivek if you'd like help as iterating 🙂

**Comment by owocki** [comment]
Thank you vivek.  Yes, thats right.

@junagao sorry for the radio silence.  Vivek hopefully just unblocked you!

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

@junagao due to inactivity, we have escalated [this issue](https://gitcoin.co/issue/gitcoinco/web/1443/607) to Gitcoin's moderation team. Let us know if you believe this has been done in error!

* [x] warning (3 days)
* [x] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by owocki** [comment]
hey @junagao hows it going?!  let me know if you need further clarification and what your timeline is

**Comment by owocki** [comment]
@junagao i am removing you from this issue.

@justpixel any interest in picking this up?  bounty is open again

**Comment by octavioamu** [comment]
Can I change the text from medium to be in plural to reflect the company instead of personal opinion, and also some sentences to be more direct to the point? 
I already selected some paragraphs but I will try to go out from the article way and be more informative. What do you guys think?

**Comment by octavioamu** [comment]
First try, is not finish but I need some feedbacks about text and style. 
What do you guys think about the text? 
Need more text? less text? more text fragmentation?  
I think since we are making arguments is hard to just put short texts like mission page I'm trying to balance between. 

![desktop hd](https://user-images.githubusercontent.com/1358093/43353391-e417e774-920d-11e8-9145-c515b0781b20.jpg)

**Comment by thelostone-mc** [comment]
@octavioamu i like the amount of text you have right now on this :D 
Can we loose the yellow buttons and stick to white /  blue (the ones we use / subtle color which matches the style on the buttons we already have ) ? :P

**Comment by owocki** [comment]
i think that we should blow away 90% of the text on this page; and aim for a text to image ratio like [this area of the page](http://bits.owocki.com/1X0s002C1E3Q/Screen%20Shot%202018-07-30%20at%2011.50.32%20AM.png) for the entire page.

you can take the content @vs77bb posted above

```
It is still early days in blockchain, including tokens / tokenization.
Gitcoin didn't want to launch with a token before finding product-market fit, and doesn't think it's the morally right move. .
There are a number of other options to explore in terms of capital in web3 (great job highlighting these!)
Founders should consider these other options, and think before doing an ICO. Make the right chess move with your time.
```

and maybe break it down into the following UI elements

# It is still early days in blockchain, including tokens / tokenization.

h1 - its the early days
(left image), right icon and h3 / paragraph text

# Gitcoin didn't want to launch with a token before finding product-market fit, and doesn't think it's the morally right move. .

h1- Gitcoin didn't want to do a utility token
(right image)
left icon and h3 / paragraph text

etc....

# There are a number of other options to explore in terms of capital in web3

(this section is already looking good)

etc

**Comment by vs77bb** [comment]
Agree with @owocki's sentiments above! Less text is more here.

**Comment by gitcoinbot** [comment]
@octavioamu Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1443/607?snooze=100)

**Comment by octavioamu** [comment]
@gitcoinbot tomorrow/today night  I will have something

**Comment by owocki** [comment]
this is <3 ! 

@PixelantDesign @vs77bb @thelostone-mc et all, any feedback?

**Comment by PixelantDesign** [comment]
Looks good!

**Comment by octavioamu** [comment]
PR done with the code and also in /creative with the sketch file https://github.com/gitcoinco/creative/pull/21

**Comment by octavioamu** [comment]
I have summited work  https://etherscan.io/tx/0x9f2e2e0f3f681f6d453b2b088e5a655417bc091d6c28a6d95c7a72cb501f29e3
but looks like @gitcoinbot doesn't want to work

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. **Submitted** 4. Done 

<hr>

__Work for 130.0 DAI (130.0 USD @ $1.0/DAI) has been submitted by__:

 1. [@octavioamu](https://gitcoin.co/profile/octavioamu)



@owocki please take a look at the submitted work:
* [PR](https://github.com/gitcoinco/creative/pull/21) by @octavioamu

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/1443/607)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/1443/607).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $29,092.82 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__The funding of 130.0 DAI (130.0 USD @ $1.0/DAI)  attached to this issue has been approved & issued to @octavioamu.__  

 * Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/1443/607)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $28,220.07 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
